MessengerOnFileForLinux/FileHandling/src/FileHandling.cpp

```diff
@@ -356,6 +356,7 @@ bool FileInterface::Modification::updateRowField(const std::string& pathToFile,
 bool FileInterface::lockFolder(const std::string& pathToFolder)
 {
     waitForAccess(pathToFolder);
+    return not FileInterface::Managment::isFileExist(pathToFolder + "/GUARD");
```

Comment:
1. FileInterface:: niepotrzebne. 
2.A może lockFolder zmienić na void, i tak nie korzystamy nigdzie z żadnego przypisania wyniku tej funkcji ?
3.Dodatkowo jak już tu jesteś, to przydałoby się zabezpieczyć funckję przed sytuacją kiedy nie udało nam się stowrzyć guarda. Bo aktualnie w waitForAccess() wywołujemy createGuard() i radośnie zakładamy że się zawsze utworzy.
4. Trzeba zrobić taska na refaktor tego pliku, bo aż mi sie niedobrze robi jak patrze na funckje wait..., która robi jeszcze w srodku milion rzeczy :-D

Reply:
1. Tak, ale trzymałem się konwencji z Twojej funkcji na dole.
2. nie void, bo wtedy nie kontrolujemy potem. Wiem, że nie korzystamy z żadnego rpzypisania bo grepowałem, ale uznałem, że wolimy mieć funkcję gotową na korzystanie z tego niż potem to jeszcze raz zmieniać.
3. To nie jest scope tego taska. Tworzę nowego taska na to.
4. Nie chcę sugerować kto to pisał :package:

Reply:
A tak wgl to jest błąd, bo nie powinno być "not" w funkcji